base/fastmath.jl

```diff
@@ -124,6 +125,38 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+@fastmath expr
```

Comment:
> Just indent everything inside the triple quotes, you mean?

No, just the signature, see [the doc for other macros/functions](https://github.com/JuliaLang/julia/blob/d3db31200c09a99ad9f7cbc019dd0aa56c6944a7/base/essentials.jl#L200)

> added to the doc/stdlib/

See https://github.com/JuliaLang/julia/blob/d3db31200c09a99ad9f7cbc019dd0aa56c6944a7/CONTRIBUTING.md#improving-documentation There are examples there that you can follow. And you can decide which file to put it in...